README.md

```diff
@@ -46,8 +46,8 @@ Important note! If you are using `@fastify/compress` plugin together with `@fast
 ### Per Route options
 You can specify different options for compression per route by passing in the `compress` options on the route's configuration.
 ```javascript
-fastify.register(
-  require('@fastify/compress'),
+await fastify.register(
+  import('@fastify/compress'),
```

Comment:
Import here should not work. It may not throw error, but the function execute is wrong.
https://github.com/fastify/avvio/blob/74334bbc0a470aeca41a42d8b30ff5cebf7a3f21/boot.js#L206-L208
```suggestion
  require('@fastify/compress'),
```

Reply:
It _actually_ works even without this PR, avvio handles  it, check it out.

Reply:
Let's start using more import in our docs when registering plugins. It is really a cool feature that avvio can handle for ages.